mkdocs.yml

```
@@ -6,6 +6,7 @@ nav:
   - Deployment: deployment.md
   - Running Locally: running-locally.md
   - Backstage Update: backstage-update.md
+  - RFC Doc Generator: rfc-doc-generator.md
```

I think we can add a folder for RFCs like this 
```suggestion
  RFCs and decision records: 
    - RFC Doc Generator: rfc-doc-generator.md
```